### PR TITLE
Bug 1206780 - Add Pontoon site name and details to Persona popup

### PR DESCRIPTION
**Pull request by m8ttyB**

### CODE REVIEW

File: pontoon/settings/base.py [hunk 1]
```diff
@@ -411,6 +411,7 @@ def _allowed_hosts():
 LOGIN_REDIRECT_URL = '/'
 LOGIN_REDIRECT_URL_FAILURE = '/'
 BROWSERID_VERIFY_CLASS = 'pontoon.base.views.BrowserIDVerify'
+BROWSERID_REQUEST_ARGS ={'siteName': 'Pontoon'}
```
Comment (Osmose) [inline pontoon/settings/base.py]: Space after the equals pleaaassssssseee

**Comment by Osmose** [comment]
r+, nice work!

**Comment by Osmose** [comment]
Merged in cc7b49a.

File: pontoon/settings/base.py (hunk 1 above)
Comment (m8ttyB) [inline pontoon/settings/base.py]: Good catch .. fixed.

**Pull request closed without merging**